test/login/components/SearchHelpSignIn.unit.spec.jsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
+import _ from 'lodash';
 
 import { SearchHelpSignIn } from '../../../src/js/login/components/SearchHelpSignIn.jsx';
 
+const defaultLoginProps = {
+  currentlyLoggedIn: false,
+  loginUrl: '',
+  utilitiesMenuIsOpen: {
+    account: false,
+    help: false,
+    search: false
+  }
+};
+
+const defaultProfileProps = {
+  email: 'fake@aol.com',
+  userFullName: {
+    first: 'Sharon'
+  }
+};
+
+function buildProps(defaultProps, props = {}) {
+  return _.merge({}, defaultProps, props);
```

Comment:
I would suggest using `merge` from `lodash/fp` if you don't want the extra arg to ensure both objects get merged into a new object. The FP version will just create a new object by default.